### PR TITLE
feat: optimize build command

### PR DESCRIPTION
**Pull request by hwbrzzl**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: console/cli_context.go [hunk 1]
```diff
@@ -229,17 +229,22 @@ func (r *CliContext) Secret(question string, option ...console.SecretOption) (st
 	return answer, nil
```
Comment (kkumar-gcc) [inline console/cli_context.go:229]: please update `cli_context_test.go`

File: console/console/build_command.go [hunk 2]
```diff
@@ -38,70 +37,99 @@ func (receiver *BuildCommand) Extend() command.Extend {
...
-		for _, validOption := range validSystems {
-			if option == validOption {
-				return true
-			}
+	os := ctx.Option("os")
+	if os == "" {
+		os, err = ctx.Choice("Select target os", []console.Choice{
+			{Key: "Linux", Value: "linux"},
+			{Key: "Darwin", Value: "windows"},
+			{Key: "Windows", Value: "darwin"},
```
Comment (rusman-plat-d) [inline console/console/build_command.go:86]: it doesn't seem to match in darwin and windows value

Comment (hwbrzzl) [reply console/console/build_command.go:86]: Good catch

**Review by kkumar-gcc — APPROVED** [review]
LGTM

**Pull request merged**